packages/backend/src/models/table-metadata.ts

```diff
@@ -53,6 +55,22 @@ class TableMetadata extends Base {
     }
     return true
   }
+
+  async stripInvalidKeys(rows: ITableRow[]): Promise<ITableRow[]> {
+    const columns = await this.$relatedQuery('columns')
```

Comment:
Small nit as per previous PR. Can `select('id')` or `withGraphJoined` to microoptimize